tracer/src/Datadog.Trace.OpenTracing/Datadog.Trace.OpenTracing.csproj

```diff
@@ -16,14 +16,15 @@
     <ProjectReference Include="..\Datadog.Trace\Datadog.Trace.csproj" />
 
     <PackageReference Include="OpenTracing" Version="0.12.1" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1' ">
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net461' ">
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="5.0.1" />
```

Comment:
`Microsoft.Extensions.Primitives` 6.0 dropped support for .NET Core 2.1, so we need to stick with 5.0.1 for now.

Reply:
I think we should probably use the _lowest_ possible version here, i.e. 2.0.0. Otherwise, customers are forced to upgrade their version of _Microsoft.Extensions.Primitives_ to 5.0.1 at a minimum, which could conflict with their other dependencies.

At most, we should prob include 2.0.0 in the netstandard2.0 target, and 3.1.0 in the 3.1 target...

Also, this is a breaking change, do we care? 🤔

Reply:
> we should probably use the lowest possible version here

Good point. Changed to 2.0.0 in 1c71eb0ea90d89a32e197d6aa9913105a0b37332.

> this is a breaking change, do we care?

How is it a breaking change?

Reply:
I removed the reference to `Microsoft.Extensions.Primitives`.